tasks/eth/zora-002-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,77 @@
+# Validation
```

Comment:
We are missing the expected domain and message hashes in this file, using the same format as other validation files. I am seeing:

- Council domain hash: 0xdf53d510b56e539b90b369ef08fce3631020fbf921e3136ea5f8747c20bce967
- Council message hash: 0xbe97205c6515952baed5c1a4fba2d6598654b5e909affa117fe9664c79498653
- Foundation domain hash: 0xa4a9c312badf3fcaa05eafe5dc9bee8bd9316c78ee8b0bebe3115bb21b732672
- Foundation message hash: 0xf715627716306b639e9c2cfe9685ff147889862c04b8f878e0d1aaaf2aa485ff